roles/kubernetes-apps/network_plugin/meta/main.yml

```diff
@@ -31,6 +31,6 @@ dependencies:
       - kube-router
 
   - role: kubernetes-apps/network_plugin/multus
-    when: kube_network_plugin_multus
+    when: kube_network_plugin_multus|default(false)|bool
```

Comment:
You don't need to change this, it's already defined in `kubespray-defaults` to false